manager/manager.go

```diff
@@ -650,6 +650,7 @@ func (m *manager) containerDataSliceToContainerInfoSlice(containers []*container
 		cinfo, err := m.containerDataToContainerInfo(containers[i], query)
 		if err != nil {
 			// Skip containers with errors, we try to degrade gracefully.
+			klog.Warningf("convert container data to container info failed with error %s", err.Error())
```

Comment:
lets be a little more cautious and add that at V(4) instead of warning to start with.